### PR TITLE
Analytics Plugin Documentation

### PR DESCRIPTION
**Pull request by joemcelroy**
Adds search-ui behavioral analytics plugin documentation

### CODE REVIEW

File: docs/api-plugins-analytics-plugin.mdx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+id: api-plugins-analytics-plugin
+slug: /search-ui/api/core/plugins/analytics-plugin
+title: Analytics Plugin
+date: 2023-01-05
+tags: [""]
+---
+
+The Analytics Plugin is a plugin that provides a way to send analytics events to the Behavioral Analytics Product. To use it, you must follow the instructions on how to set up the Analytics Product.
```
Comment (leemthompo) [inline docs/api-plugins-analytics-plugin.mdx:9]:
```suggestion
Use the Analytics Plugin to send analytics events to the Behavioral Analytics Product. Follow the instructions to set up the plugin.
```

Minor rewording for simplicity

File: docs/api-plugins-analytics-plugin.mdx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+```bash
+yarn add @elastic/search-ui-analytics-plugin
+## OR
+npm install @elastic/search-ui-analytics-plugin
+```
+
+## Basic Usage
+
+Within your Search UI configuration, you can add the Analytics Plugin like so:
```
Comment (leemthompo) [inline docs/api-plugins-analytics-plugin.mdx:21]: Is there a missing example here after "like so:"?

File: docs/api-plugins-analytics-plugin.mdx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+By default, the Analytics Plugin will use the Behavioral Analytics client thats provided when the integrated via script tag.
```
Comment (leemthompo) [inline docs/api-plugins-analytics-plugin.mdx:23]:
```suggestion
By default, the Analytics Plugin will use the Behavioral Analytics client provided when integrated using a `script` tag.
```

**Review by leemthompo — DISMISSED** [review]
@joemcelroy Looks good! I just made a few some minor cosmetic rewording suggestions. Otherwise, there does look to be one missing code sample.

The repo README looks to be a duplicate of the MDX file, so you can just assume the same comments apply to that file 😉.

File: docs/api-plugins-analytics-plugin.mdx (hunk 2 above)
Comment (joemcelroy) [reply docs/api-plugins-analytics-plugin.mdx:21]: the example is below doesn't follow immediately after. Ive updated it to be immediately after :)

**Comment by elasticdocs** [comment]
:rocket: Built _docs-elastic-co_ successfully!

* https://docs-elastic-7m6u806cv-elastic-dev.vercel.app

* Built with commit bb9050afad64687c32219b6064aca47ad19d1d8d

> Issues? Visit #next-docs in Slack

**Review by leemthompo — APPROVED** [review]
👍

**Pull request merged**